feat(card): added new component

agliga opened the pull request:
Fixes #74

- [ ] This PR contains CSS changes
- [X] This PR does not contain CSS changes

## Description
* Added new card component
* Added new marko 6 component (note, tests in marko 6 are not being run yet)

## Checklist
- [X] I verify the build is in a non-broken state
- [X] I verify all changes are within scope of the linked issue

LuLaValva commented on packages/evo-marko/src/tags/evo-card/index.marko:
```diff
@@ -0,0 +1,80 @@
+    tabindex=disabled && -1
+    ...htmlInput>
+    <div class="card__media">
+        <img ...file>
```
What happens here if this is actually a [File](https://developer.mozilla.org/en-US/docs/Web/API/File/File), as suggested in the `Input` type? I don't think it makes sense to spread that into `<img>`

HenriqueLimas replied:
Should we do something similar that we do on [file-preview-card](https://github.com/eBay/evo-web/blob/main/packages/ebayui-core/src/components/ebay-file-preview-card/index.marko#L44-L59) where we check for the type and render accordingly?

LuLaValva reviewed (commented):
Just one thing I'm confused about, but otherwise looks good

HenriqueLimas commented on packages/ebayui-core/src/components/ebay-card/index.marko:
```diff
@@ -0,0 +1,94 @@
+    ...processHtmlAttributes(htmlInput)
+>
+    <div class="card__media">
+        <img ...processHtmlAttributes(file || [])>
```
`file` is not a required type, should we render `card__media` only when `file` is present?

agliga replied:
I will swap this to image attribute tag. I think its cleaner

agliga commented on packages/ebayui-core/src/components/ebay-card/examples/anchor.marko:
```diff
@@ -0,0 +1,19 @@
+    }
+    ...input
+>
+    <@overline text="Authenticity Guaranteed">
```
I think the reason why this is like this is because there was a `card__overline-text` class being applied.
When I saw it initially, I thought this had styles associated with it. But looking again, there are no styles associated with it, so I will do this suggestion.

agliga commented:
I swapped out of using file to just using `@image` directly like I did with the chips PR
Also addressed all comments. 

I was initially going to use file preview card but there were some issues integrating with it, so that was left over.
